poetry.lock

```diff
@@ -7387,4 +7387,8 @@ vectordb = ["faiss-cpu", "numpy"]
 [metadata]
 lock-version = "2.0"
 python-versions = "^3.9"
+<<<<<<< HEAD
 content-hash = "4efb246e4ff31c3f9586aadbc21503777099be71b56e539b2c50870489aff6b1"
+=======
+content-hash = "eb8fff0171f3a678dffc37f25f04fa97f1bd13d7879562999e0c4ffc509151e3"
+>>>>>>> main
```

Comment:
Looks like a merge conflict got through.  Should be able to accept either one then just override with `poetry lock --no-update`.